rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -161,7 +173,11 @@ public void initializeServerCapabilities(ServerCapabilities result) {
         result.setImplementationProvider(true);
         result.setSemanticTokensProvider(tokenizer.options());
         result.setCodeLensProvider(new CodeLensOptions(false));
-        result.setExecuteCommandProvider(new ExecuteCommandOptions(Collections.singletonList(BaseWorkspaceService.RASCAL_META_COMMAND)));
+        String commandName = BaseWorkspaceService.RASCAL_META_COMMAND;
+        if (!dedicatedLanguageName.isEmpty()) {
+            commandName += "-" + dedicatedLanguageName;
+        }
+        result.setExecuteCommandProvider(new ExecuteCommandOptions(Collections.singletonList(commandName)));
```

Comment:
I'm missing why we check for prefix when the commandname comes back in a message from the LSP. Here we register the commandName as-is; not its prefix? pls explain.

Reply:
This is for the `rascal-meta` command feature, where a vs code extension can call a rascal registered command from "the outside". Our current implementation registers `rascal-meta-command`, so we can't have collisions there with "dedicated"  versions of this server. This was the only point where I've had to special case the code for these dedicated instances.

The non-dedicated behaves as before, it's just that in case you are in a dedicated setup, you have to invoke the command with the right suffix (namely, the language identifier, so for example `rascal-meta-command-pico`).